assets/js/scenes.js

```diff
@@ -153,88 +153,103 @@ const generateScenes = () => {
 
     generateFloorTiles();
 
-    //Generate tiles for walls
-
     const generateWallTiles = () => {
       const tileWidth = 34;
       const tileHeight = 34;
       const screenWidth = 800;
       const screenHeight = 600;
     
-      // Generate tiles along the top border
-      for (let x = 0; x < screenWidth; x += tileWidth) {
-        add([
-          sprite("wall_tile"),
-          pos(x, 0),
-          scale(1),
-          layer("bg"),
-        ]);
+      // Generate tiles along the top border (excluding specific columns)
+      for (let x = tileWidth; x < screenWidth - tileWidth; x += tileWidth) {
+        if (x !== tileWidth * 12 && x !== tileWidth * 13 && x !== tileWidth * 14 && x !== tileWidth * 15) {
+          add([
+            sprite("wall_tile"),
+            pos(x, 0),
+            scale(1),
+            area({ width: tileWidth, height: tileHeight }), 
+            layer("bg"),
+          ]);
+        }
       }
     
-      // Generate tiles along the bottom border
-      for (let x = 0; x < screenWidth; x += tileWidth) {
-        add([
-          sprite("wall_tile"),
-          pos(x, screenHeight - tileHeight),
-          scale(1),
-          layer("bg"),
-        ]);
+      // Generate tiles along the bottom border (excluding specific columns)
+      for (let x = tileWidth; x < screenWidth - tileWidth; x += tileWidth) {
+        if (x !== tileWidth * 12 && x !== tileWidth * 13 && x !== tileWidth * 14 && x !== tileWidth * 15) {
+          add([
+            sprite("wall_tile"),
+            pos(x, screenHeight - tileHeight),
+            scale(1),
+            area({ width: tileWidth, height: tileHeight }), 
+            layer("bg"),
+          ]);
+        }
       }
```

Comment:
Suggestion (Non-blocking): You've used similar code above. Do you think it would be worth refactoring it and putting it into its own function? It might be cleaner.